tests/utests/utests.h

```diff
@@ -844,16 +864,17 @@ struct utest_context {
  * @param[in] CANNONICAL_VAL expected cannonical value
  * @param[in] VALUE          expected array of bits names
  */
-#define CHECK_LYD_VALUE_BITS(NODE, CANNONICAL_VAL, VALUE) \
+#define CHECK_LYD_VALUE_BITS(NODE, CANNONICAL_VAL, ...) \
     assert_non_null((NODE).canonical); \
     assert_string_equal((NODE).canonical, CANNONICAL_VAL); \
     assert_non_null((NODE).realtype); \
     assert_int_equal(LY_TYPE_BITS, (NODE).realtype->basetype); \
     { \
-        LY_ARRAY_COUNT_TYPE arr_size = sizeof(VALUE) / sizeof(VALUE[0]); \
+        const char *arr[]={ __VA_ARGS__ };\
```

Comment:
Strictly adhering to the coding style, missing spaces around `=` and before the last `\`.